### PR TITLE
Add LaunchDarkly to /health API endpoint

### PR DESCRIPTION
**Pull request by mojotalantikite**
## Summary
This adds LD to the `/health` api endpoint. It includes the appropriate secrets for each environment and is mainly for us to test enabling/disabling something via LD.

#### Related issues
https://qmacbis.atlassian.net/browse/MR-402

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-    "eslint.workingDirectories": ["services/app-web", "services/app-api"]
+    "eslint.workingDirectories": ["services/app-web", "services/app-api"],
+    "launchdarkly.project": "default",
```
Comment (macrael) [inline .vscode/settings.json:3]: Is there a VSCode plugin related to LD?

**Review by macrael — APPROVED** [review]
Cool!

File: .vscode/settings.json (hunk 1 above)
Comment (mojotalantikite) [reply .vscode/settings.json:3]: There is, it let's you do things like see flag toggle state and whatnot from VSCode: https://docs.launchdarkly.com/integrations/vscode#installing-the-vscode-extension

File: services/app-api/handlers/health_check.ts [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+import LaunchDarkly from 'launchdarkly-node-server-sdk'
+
+const ldClientKey = process.env.LD_SDK_KEY ?? ''
+if (ldClientKey === '') {
+    throw new Error('LD_SDK_KEY environment variable is not set')
+}
 
 export const main: APIGatewayProxyHandler = async () => {
+    const ldClient = LaunchDarkly.init(ldClientKey)
+    await ldClient.waitForInitialization()
```
Comment (rswerve) [inline services/app-api/handlers/health_check.ts:11]: Do we know what happens here if LD doesn't respond?

Comment (mojotalantikite) [reply services/app-api/handlers/health_check.ts]: Ah yeah, it'll promise reject -- I just pushed a change to try/catch it.

**Pull request merged**